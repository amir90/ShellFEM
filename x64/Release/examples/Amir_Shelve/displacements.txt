-8.23877e-08 -1.50912e-05 -1.99175e-06
 -4.1211e-08 -6.42136e-06 -1.99173e-06
-7.20132e-11  5.31815e-07 -1.99165e-06
           0            0            0
           0            0            0
-8.23763e-08 -1.50669e-05 -2.03296e-06
 -4.1209e-08 -6.46783e-06 -2.03298e-06
 -9.1423e-12  5.42999e-07 -2.03305e-06
           0            0            0
           0            0            0
-8.23337e-08 -1.50242e-05 -2.07412e-06
-4.11711e-08 -6.43076e-06 -2.07413e-06
-7.11217e-12  5.53984e-07 -2.07414e-06
           0            0            0
           0            0            0
-8.23091e-08 -1.52387e-05 -2.11525e-06
-4.11576e-08 -6.53117e-06 -2.11524e-06
 9.52264e-13  5.64908e-07 -2.11523e-06
           0            0            0
           0            0            0
-8.23058e-08 -1.56442e-05 -2.15639e-06
-4.11545e-08 -6.75409e-06 -2.15638e-06
 3.88524e-11  5.75915e-07 -2.15638e-06
           0            0            0
           0            0            0