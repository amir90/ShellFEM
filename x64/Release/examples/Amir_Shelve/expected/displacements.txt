0 -7.1010653e-10 0.00035935405 -0.0013447414
1 5.2672914e-07 -0.01224247 -0.0013436853
2 6.4306566e-10 0.00035907168 -0.0013436856
3 5.283815e-07 -0.011798584 -0.0013447402
4 6.3853354e-38 -2.4633493e-37 1.3161735e-37
5 -1.6470105e-37 -2.0289035e-37 -6.604306e-38
6 -0.0 -0.0 0.0
7 -0.0 -0.0 0.0
8 -1.337439e-37 -5.325946e-38 -2.8655639e-37
9 0.0 -0.0 -0.0
10 1.4216796e-10 0.0003592134 -0.0013442091
11 5.273546e-07 -0.01178138 -0.0013442106
12 1.0679116e-37 -7.900874e-37 2.9767082e-37
13 1.2780044e-37 -7.0742286e-37 -7.668874e-38
14 0.0 -0.0 0.0
15 0.0 -0.0 -0.0
16 2.637162e-07 -0.004870697 -0.0013436854
17 2.6386402e-07 -0.0047147744 -0.0013447403
18 1.9748331e-10 0.00035914002 -0.0013439495
19 -8.716969e-10 0.00035927998 -0.0013444743
20 5.281953e-07 -0.011746097 -0.001344475
21 5.266239e-07 -0.011956365 -0.0013439483
22 2.6369085e-07 -0.004725129 -0.0013442101
23 2.6341098e-07 -0.00479441 -0.0013439485
24 2.6396606e-07 -0.0047014225 -0.0013444747
